vocabularies/voc4cat/0007014.ttl

```diff
@@ -0,0 +1,14 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0007014>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0007014"^^xsd:token ;
+    skos:broader <https://w3id.org/nfdi4cat/voc4cat_0007003> ;
+    skos:definition "Some characteristics that define the catalyst,which  can be related to composition, active phase, role. Examples are bulk catalyst, supported catalyst, oxide catalyst, metal catalyst, model catalyst, …"@en ;
```

Comment:
- A space should be added between: ... the catalyst**,**which...
- Maybe the examples are not necessary in the definition. We can add them as separate concepts if needed. For instance metal oxide exists already (nfdi4cat/voc4cat_0000005).

Reply:
good